Compiler/FunctionCompiler.py

```diff
@@ -19,19 +19,7 @@
 """
 
 
-class LibraryFunctionCompiler:
```

Comment:
I think this class belongs in the same file as the other function compiler, and not in Globals